### DISCUSSION TITLE
Setting tlsMode while deploy using helm chart

### DISCUSSION THREAD

**Discussion by Bhautik0110** [zitadel/zitadel#5036] [Q&A]
How can I set `tlsMode` external when I deploy zitadel using helm chart?

**Comment by BhautikChudasama** [comment]
Hey @fforootd, Is that anyway?  Do I need to deploy using the plain manifest file?

**Reply by fforootd** [reply]
You can extend the values.yaml with the needed settings 

https://github.com/zitadel/zitadel-charts/blob/main/charts/zitadel/values.yaml#L6

Checkout the correct settings for the TLS mode here https://zitadel.com/docs/self-hosting/manage/tls_modes
